### ISSUE TITLE
A credit list for people that have contributed to the RedMew repository

### ISSUE THREAD

**Issue by iltar** [Valansch/RedMew#282]
I think it would be nice to create a list of people who contributed to the RedMew code-base. Whether it's a typo fix, a balance change, bug fix or a whole new feature. This might encourage players to contribute if they wish, even if it's just to say "Hey, I've contributed to RedMew!". Could be displayed via a GUI in-game for example.

**Comment by SimonFlapse** [comment]
We could add it at the bottom of the "Welcome" tab in the info GUI.

**Comment by iltar** [comment]
That's not a bad idea, or perhaps a new tab? Would prefer to keep the names limited to factorio names

**Comment by SimonFlapse** [comment]
![image](https://user-images.githubusercontent.com/44922798/48314452-0b449a80-e5ca-11e8-8ef0-a3858d96104d.png)

I like the looks of the info GUI as it is. So I don't think a new tab would be a good idea.

My suggestion would still be to do it at the bottom of the "Welcome" tab, or rethink the "Other Servers" tab which hasn't been used.

Side note: The "How To Chat" seems to be placed out of sight for the players that really needs it, I'm not sure what to do about it.

**Comment by plague006** [comment]
I think listing 17 names in a GUI window that is already teeming with information that already goes unread is just adding extra noise. At most we could link to the contributors list somewhere: https://github.com/Valansch/RedMew/graphs/contributors

**Comment by iltar** [comment]
My github name is not the same as my in-game name, so that doesn't seem feasible to me. It should be visible somewhere in-game, with player names, even if it's not obvious to be found.

**Comment by plague006** [comment]
I just fundamentally disagree. As things currently stand: questions are asked hourly about information that is provided in 'info'. We should be trying to trim down 'info' to funnel people towards the most important information.

**Comment by plague006** [comment]
I just fundamentally disagree. As things currently stand: questions are asked hourly about information that is provided in 'info'. We should be trying to trim down 'info' to funnel people towards the most important information.

**Comment by SimonFlapse** [comment]
We could have the info window open automatic on first join

**Comment by Jayefuu** [comment]
Seems like make-work to me. Isn't that what the contributors tab on Github is for?

**Comment by iltar** [comment]
You won't find me in-game under this name, which is what I'm trying to point out

**Comment by plague006** [comment]
Yeah that's a tough issue... I totally get wanting recognition for the work. It's just a question of implementing it in a way that doesn't force it on people but let's them find the info if they want it.

**Comment by SimonFlapse** [comment]
We have a lot going space in the info GUI. Most people don't go there anyways, but I'm sure it can be added. Only problem is keeping it up to date. Right now maps are tagged with credits for the players who made the map, but I understand that for diggy that list might become long

**Comment by theorangeangle** [comment]
Maybe as a compromise we have a "show contributors" button in the other servers tab? And rename it to "other info". 


Also, if we are putting it somewhere infrequently read then making it an easter egg might be a nice idea (give the player a fish if they open the contributors tab or something)

**Comment by linaori** [comment]
Or coins? 😄

**Comment by mheguy** [comment]
> in the other servers tab

Should... should we tell him?

On a more serious note, I think `new_info_key` is completely neglected, we could repurpose that whole tab for a contributors list.

Also.. completely unironically, I love the idea of coins for looking at the info tabs. It's economics 101: give an incentive for actions you want to promote.

**Comment by theorangeangle** [comment]
Similar to how we name outposts in crash site after donators, how about naming rockets after github contributors? Name the first rocket launched as "Rocket dedicated to contributor [x]" in chat instead of "a rocket has been launched", with people who contribute more getting higher up on the launch list. After all contributors have been named, chat messages return to the default/current message.

**Comment by mheguy** [comment]
I like that. I want to change the idea but I like it overall.

**Comment by grilledham** [comment]
I think having a tab in the info gui for supporters is a good idea.

We could auto populate the list of supports form the donator scenario dataset. That would probably mean the least maintenance effort from us.

I'm not sure how we would add github contributors to that though. Either we add them to the donator dataset, make a new dataset, add a file in the repo or ignore them and make this just about patreon donators.
